jpsreport/methods/Method_A.cpp

```diff
@@ -77,9 +73,8 @@ bool Method_A::Process(
     Log->Write("------------------------Analyzing with Method A-----------------------------");
     //for(int frameNr = 0; frameNr < peddata.GetNumFrames(); frameNr++ )
     bool PedInGeometry = false;
-    for(std::map<int, std::vector<int>>::iterator ite = _peds_t.begin(); ite != _peds_t.end();
-        ite++) {
-        int frameNr = ite->first;
+    for(auto ite : _peds_t) {
+        int frameNr = ite.first;
```

Comment:
You could use structured bindings instead:
```
for (const auto [frameNr, _] : _peds_t){
...
}
```

Reply:
I rolled back this change since I don't want to do any refactoring at this stage.